Sources/MuxPlayerSwift/FairPlay/FairPlaySessionManager.swift

```diff
@@ -281,7 +308,7 @@ class DefaultFairPlayStreamingSessionManager<
             guard let data = data,
                   data.count > 0
             else {
-                print("No CKC data despite server returning success")
+                self.logger.debug("No CKC data despite server returning success")
```

Comment:
nit: perhaps `.fault` or `.error`?

Reply:
Yeah one of those makes more sense. Will address in separate PR.